spec/models/vim_performance_state_spec.rb

```diff
@@ -20,6 +20,49 @@
     end
   end
 
+  describe "#containers" do
+    let(:capture_time) { 5.hours.ago.utc.beginning_of_hour }
+
+    it "doesnt return records that are archived before the range" do
+      container_image = FactoryBot.create(:container_image)
+      containers, other_containers = create_past_present_future(:containers, {:container_image => container_image}, return_old: true)
+      state_data = {:assoc_ids => {:containers => {:on => (containers + other_containers).map(&:id)}}}
```

Comment:
Super minor, but leave a blank line here to match the pattern of the test below
```suggestion
      containers, other_containers = create_past_present_future(:containers, {:container_image => container_image}, return_old: true)

      state_data = {:assoc_ids => {:containers => {:on => (containers + other_containers).map(&:id)}}}
```